### ISSUE TITLE
An urgent need

### ISSUE THREAD

**Issue by kilanoff** [mattermost/mattermost#29636]
Hello! 
It is not very convenient to use MM

Therefore, I would like the following:

1. An indicator of new messages in the form of a red dot on a channel or person, as in any messenger
2. Android App: In the application, in order to send a message, you have to click twice on the send button, apparently this is due to the fact that first the focus falls on the block with the button, and then only the button can be pressed
3. There are not found a voice messages in the app
4. In order to select an emoticon or reaction, you need to do a lot of actions in the application. It would be convenient if the emoticon insertion button was accessible on the message write panel as in any of the popular messengers
5. There is a great lack of styling of message blocks or a change of design theme, so that message blocks stand out visually and do not merge into a single text.

Sorry if I'm repeating myself with other requests

**Comment by vtochenyy** [comment]
I agree wih you. In mm i face such problems every day. It's not very convenient!

**Comment by amyblais** [comment]
Hi @kilanoff, can you help fill in more details as requested in the New Issue template?

#### Summary
Bug report in one concise sentence.

#### Steps to reproduce
What are the steps to reproduce the issue?

#### Operating System
What Operating System do you use? (examples: Windows 10 x64, macOS Ventura 13.2 Apple Silicon, Ubuntu Linux 22.04 LTS x64)

#### Mattermost Server Version
Which version of the Mattermost Server did this occur on? (You can find your Mattermost Server version via **Mattermost Menu** > **About Mattermost**)

#### Expected behavior
What is the expected behaviour?

#### Observed behavior
What issue did you see happen?

#### Log Output
Output from the log files if applicable.

#### Additional Information
Additional helpful information, such as error messages and/or screenshots.

#### Possible fixes
If you can, link to the line of code that might be responsible for the problem.

**Comment by kilanoff** [comment]
according to point 2

Summary
The problem is in the quick click of the send button. It takes two clicks for the message to be sent

Steps to reproduce
Appears in any chat where there is a send message

Operating System
Samsung Galaxy s24 Ultra, One UI 6.1, Android 14

Mattermost Server Version
I don't know the version

Expected behavior
Waiting - instant sending of a message at the first click on the "Send" button
but, I can't reproduce the problem at the moment :(

Observed behavior
The problem is that the first time you press the button, the message is not sent, you need to press the button again
but, I can't reproduce the problem at the moment :(

Log Output
no data

Additional Information
just a send message button (like a paper airplane)

Possible fixes
no code link

but I had a problem and it appears periodically
Thanks.

**Comment by amyblais** [comment]
Thank you, you can find the Mattermost server version and also the Mattermost mobile app version via Main Menu -> Settings -> About Mattermost.

**Comment by NadavTasher** [comment]
Mattermost is an open-source project, meaning if there's something you don't like, you have the power to fix it yourself and even submit a PR to improve it for everyone.

You’ve raised some good points, and it might be more effective to open separate issues for each feature request. You could also contribute by writing the code yourself or engaging with maintainers and contributors at https://community.mattermost.com.
